Scripts/Runtime/Entities/MemsetBufferJob.cs

```diff
@@ -13,6 +16,36 @@ namespace Anvil.Unity.DOTS.Entities
     [BurstCompile]
     public struct MemsetBufferJob<T> : IJobParallelForBatch where T : struct, IBufferElementData
     {
+        // The minimum number of elements to process in a single batch.
+        // Attempts to keep batch sizes large enough to prevent multiple cores from copying the same
+        // data into cache. This job's operation is so simple the copy into cache is probably the most expensive part.
+        //
+        // Ex: We shouldn't split a set of 10 across 5 threads.
+        //
+        // Re CacheSize: If generalizing to a 64kb cache size is good enough for Unity it's good 
+        // enough for us.
+        private static readonly float MIN_BATCH_SIZE_PER_THREAD = math.max(1f, JobsUtility.CacheLineSize * 1024 / (float)UnsafeUtility.SizeOf<T>());
```

Comment:
I don't think this is quite correct. 

The `CacheLineSize` is 64 bytes but we're multiplying arbitrarily by 1024 to get 64 kilobytes. The L1 cache on most CPU's is at least 64 kilobytes but could be different as you and I have discussed. 

If we feel it's important to take up L1 cache we could calculate the real L1 cache size via:
https://stackoverflow.com/questions/6995787/how-to-determine-cpu-cache-size-in-net

But that might be overkill. 

If we feel it should be 64 kilobytes, then I think we should change the calculation to be `64 * 1024` and eliminate the dependency on the `JobsUtility.CacheLineSize`. They aren't really related for what we're doing it's just a happy accident. 

In either case, the comment about 64kb being good enough for Unity isn't quite right because Unity uses 16kb chunk sizes. 

This thread has more info, starting with the direct linked comment: https://forum.unity.com/threads/ok-ecs-works-in-16k-chunks-but-what-is-the-instruction-size-limit-in-ecs.577894/#post-3860092

In the code you can see in `Chunk.kChunkSize` where it's declared as a const of `16 * 1024`. Sadly it's not public because `Chunk` is internal where we can just grab it. 

Looking for `JobsUtility.CacheLineSize` that's used internally by Unity primarily for sizing structs in the collections so that as you iterate over buckets, the data for each bucket occupies it's own cacheline. (I'll confess I'm not 100% sure the reason for this) But it has no relation to Unity `Chunk` size or the actual CPU `L1 Cache` size.